include/json/reader.h

```diff
@@ -324,6 +324,8 @@ class JSON_API CharReaderBuilder : public CharReader::Factory {
    * - `"allowSpecialFloats": false or true`
    *   - If true, special float values (NaNs and infinities) are allowed and
    *     their values are lossfree restorable.
+   * - `"skipBom": false or true`
+   *   - If true, if the input starts with the Unicode byte order mark (BOM), it is skipped.
```

Comment:
This line is wider than the 80-column convention. We can take out a little redundancy and shorten it, or wrap before 80 columns.

```suggestion
   *   - If true, skip over any leading Unicode byte order mark (BOM).
```